source/DistanceAndDirection/ProAppDistanceAndDirectionModule/ProAppDistanceAndDirectionModule/ViewModels/ProTabBaseViewModel.cs

```diff
@@ -821,8 +822,9 @@ private void SetGeodesicDistance(MapPoint p1, MapPoint p2)
             });
 
             ClearTempGraphics();
+            Geometry newline = GeometryEngine.GeodeticDensifyByLength(polyline, 0, LinearUnit.Meters, type);
```

Comment:
Could maybe add a comment about what the default value ("0") does (I guess says the max segment length will be 50km: https://pro.arcgis.com/en/pro-app/sdk/api-reference/#topic8245.html )